tests/e2e/specs/tree.js

```diff
@@ -0,0 +1,16 @@
+describe('Tree component', () => {
+  it('Should display two cycle points for the mocked workflow', () => {
+    cy.visit('/#/workflows/one')
+    cy
+      .get('.node-data-cyclepoint')
+      .should(($div) => {
+        // by default, in our expected viewport size for tests, both cycle points exist and are visible
+        expect($div).to.have.length(2)
+        expect($div.get(0)).to.contain('20000101T0000Z')
+        expect($div.get(1)).to.contain('20000102T0000Z')
+      })
+  })
```

Comment:
These are the first e2e tests for the tree component :tada: which I hope will prove useful not only for the offline mode, but also pretty soon for the infinite tree. So that when merging we know it still works the same way (I hope I won't have to touch any tests for the infinite tree, as I am adding simple tests using the visible nodes only)